dogs.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <title>Adopt A Dog</title>
+</head>
+<body>
+  <div class="grid-container">
+      <section id="banner">
+          <div class="filler">
+            </div>
+        <div id="letterhead">
+          <img src="images/adoptadog-logo.svg" id="toplogo" alt= "profile of a dog in white">
+          Seattle's premiere dog adoption agency
+        </div>
+        <div id="cart">
+          <div class="top-cart">
+            <img src="images/cart-icon.svg" id="carticon" alt="a white cart logo">
+            &nbsp;&nbsp;$1234.56
+          </div>
+          <div class="checkout-button">
+            Checkout
+          </div>
+        </div>
+        <div class="filler">
+          </div>
+      </section>
+
+    <section id="subbanner">
+      <div >
+        <p><a class="subtitle" href="index.html">Home&nbsp;&nbsp;/</a><a class="subtitle" href="dogs.html">&nbsp;&nbsp;Dogs&nbsp;&nbsp;/</a><a class="subtitle" href="blog.html">&nbsp;&nbsp;Blog</a></p>
+      </div>
+    </section>
+    <section id="welcome">
+      <div>
+        <h2>Here are all of our dogs!</h2>
+      </div>
+    </section>
+
+    <section class="profiles">
+      <div class="profile-card">
+        <div>
+          <img src="images/murphy-card.jpg" class="profilepicture" alt="one of our many wonderful dogs">
+        </div>
+        <div>
+          <h3>Murphy</h3>
+          <p class="cost"><b>Cost to Adopt:</b>&nbsp;$123.45</p>
+          <p class="bio">Fusce vel elementum dolor. Nam dictum, purus nec commodo sodales, purus ligula faucibus felis, rhoncus interdum tellus libero ut leo. Ut pharetra magna</p>
+        </div>
+        <div class="adopt-button">
+          <a class="checkout" href="checkout.html">Adopt</a>
+        </div>
+      </div>
+      <div class="profile-card">
+          <div>
+            <img src="images/poppy-card.jpg" class="profilepicture" alt="one of our many wonderful dogs">
+          </div>
+          <div>
+            <h3>Poppy</h3>
+            <p class="cost"><b>Cost to Adopt:</b>&nbsp;$123.45</p>
+            <p class="bio">Fusce vel elementum dolor. Nam dictum, purus nec commodo sodales, purus ligula faucibus felis, rhoncus interdum tellus libero ut leo. Ut pharetra magna</p>
+          </div>
+          <div class="adopt-button">
+            <a class="checkout" href="checkout.html">Adopt</a>
+          </div>
+        </div>
+      <div class="profile-card">
+        <div>
+          <img src="images/jack-card.jpg" class="profilepicture" alt="one of our many wonderful dogs">
+        </div>
+        <div>
+          <h3>Jack</h3>
+          <p class="cost"><b>Cost to Adopt:</b>&nbsp;$123.45</p>
```

Comment:
instead of using the b tag, use strong and style it with css, you'll have more control